libs/common/src/vault/services/cipher.service.ts

```diff
@@ -2052,4 +2052,22 @@ export class CipherService implements CipherServiceAbstraction {
 
     return Promise.resolve(c);
   }
+
+  /**
+   * Updates the favorite status of a cipher and persists it to the server.
+   * @param id The cipher ID.
+   * @param favorite The new favorite status.
+   */
+  async updateFavorite(id: string, favorite: boolean): Promise<void> {
+    const userId = await firstValueFrom(this.accountService.activeAccount$.pipe(map((a) => a?.id)));
```

Comment:
🎨 There is a utility to get the Id by the auth team: [getUserId](https://github.com/bitwarden/clients/blob/d06d47e26a5d5306a67536cb4e89a2979e0edae9/libs/common/src/auth/services/account.service.ts#L69)


```suggestion
    const userId = await firstValueFrom(this.accountService.activeAccount$.pipe(getByUserId);
```